l10n_it_fatturapa/bindings/FatturaPA_versione_1.2.xsd.diff

```diff
@@ -0,0 +1,89 @@
+--- Schema_del_file_xml_FatturaPA_versione_1.2.xsd.orig	2019-02-12 02:28:11.342236082 +0100
```

Comment:
Quindi questo sarebbe il diff con https://www.fatturapa.gov.it/export/fatturazione/sdi/fatturapa/v1.2.1/Schema_del_file_xml_FatturaPA_versione_1.2.1.xsd giusto?

Reply:
in realtà con la 1.2 ma si aggiorna agevole

Reply:
ma è sbagliato l'xsd! c'è `<xsd:import` invece di `<xs:import`

Reply:
Perchè `xsd:import` sarebbe sbagliato?

Reply:
non c'è nessuna definizione di namespace xsd sopra